swiftide-integrations/src/openai/mod.rs

```diff
@@ -9,6 +9,10 @@ mod chat_completion;
 mod embed;
 mod simple_prompt;
 
+// expose type aliases to simplify downstream use of the open ai builder invocations
+pub type OpenAIConfig = async_openai::config::OpenAIConfig;
+pub type AzureConfig = async_openai::config::AzureConfig;
```

Comment:
You should be able to do `pub use async_openai::config::{OpenAIConfig,AzureConfig}`, then rust/docs will also recognize it as a re-export